packages/apps/esm-primary-navigation-app/src/components/choose-locale/change-locale.resource.tsx

```diff
@@ -26,15 +26,29 @@ export async function postUserPropertiesOnline(
   window.location.reload();
 }
 
+export async function postUserPropertiesOffline(
+  userUuid: string,
+  userProperties: any
```

Comment:
```suggestion
  userProperties: Record<string, string>;
```

https://rest.openmrs.org/#create-a-user